starknet_forge_template/Scarb.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "{{ PROJECT_NAME }}"
+version = "0.1.0"
```

Comment:
Nit, perhaps this should be `0.0.0` just to better show that this is a placeholder value?

Reply:
this file should be removed